src/data-source/DataSource.ts

```diff
@@ -645,6 +646,17 @@ export class DataSource {
         const flattenedEntities = ObjectUtils.mixedListToArray(
             this.options.entities || [],
         )
+
+        // resolve globs to the respective entity classes
+        flattenedEntities.push(
```

Comment:
doesn't look like a proper solution. We already do import classes from directories above by calling `buildEntityMetadatas` method.

Reply:
Ah okay! Thanks for the clarification 😄
Probably need to look through the codebase more.